### ISSUE TITLE
Drill into "for configuration" some more pls

### ISSUE THREAD

**Issue by paul-hammant** [Microsoft/schemy#6]
"for configuration" sounds like it be an alternative to YAML or TOML for config files for applications.

If yes, then an example in the README would be good.

**Comment by kflu** [comment]
Good point. Thanks. Providing a simplist example as you suggested is definately useful. Meanwhile, [here](https://github.com/Microsoft/schemy/tree/master/src/examples/command_server) is a more complex example - let me know if that helps.

**Comment by paul-hammant** [comment]
I'll circle back another day for something, I'm sure.

Meanwhile, if you've a coffee-break amount of time coming up, you may enjoy https://paulhammant.com/2013/03/28/interface-builders-alternative-lisp-timeline/ :)

**Comment by kflu** [comment]
Thanks Paul. It was an interesting read - it's encouraging to see Lisp got used in this way so many years ago 😄

**Comment by paul-hammant** [comment]
The stroke of genius was that the same script was 'played' into one of two implementations: 1 - the UI designer tool, 2 - the runtime lib (presumably callable from other languages like Objective-C).

Point being, a "smarter" editor is possible for you too. Smarter than an AST sitting behind a CR-delimited text file :)

**Comment by vivainio** [comment]
Would be nice to see an example on creating some tree-value pairs (with some lists and dicts thrown in), and reading those in in c# side

**Comment by syeerzy** [comment]
i think when we call a lisp  "for configuration" sounds like it be like emacslisp or autolisp.  

not json/YAML/TOML . not for some tree-value or pair,  but some complex things.
